### PR TITLE
Add ability to show only diff images in the html-report

### PR DESCRIPTION
**Pull request by sipayRT**
Show all images (by default): 

gemini-1

Show only diffs:

gemini-2

### CODE REVIEW

File: lib/reporters/html/static/report.css [hunk 1]
```diff
@@ -37,12 +37,25 @@
...
+    border-color: #cebe7d;
+}
+
 .image-box {
     padding: 5px;
     border: 1px solid #ccc;
     background: #c9c9c9;
 }
 
+.image-box__exp-with-act {
```
Comment (eGavr) [inline lib/reporters/html/static/report.css]: exp-with-act ?

Comment (sipayRT) [inline lib/reporters/html/static/report.css]: `image-box__expected-with-actual` - слишком длинный класс

Comment (eGavr) [inline lib/reporters/html/static/report.css]: зато понятнее :)

**Pull request merged**